### ISSUE TITLE
Error when trying to create dat files without input folder.

### ISSUE THREAD

**Issue by ArnaudBelcour** [AuReMe/mpwt#61]
If the user uses the option to create only dat files: `mpwt --dat --md -o output_folder -v`

This will lead to an error:
```Traceback (most recent call last):
  File "/usr/local/bin/mpwt", line 11, in <module>
    load_entry_point('mpwt', 'console_scripts', 'mpwt')()
  File "/mnt/c/Users/Arnaud/Downloads/Work_directory/programs/mpwt/mpwt/__main__.py", line 149, in run_mpwt
    verbose=verbose)
  File "/mnt/c/Users/Arnaud/Downloads/Work_directory/programs/mpwt/mpwt/mpwt_workflow.py", line 217, in multiprocess_pwt
    input_folder_path = input_folder + '/' + dat_run_id + '/'
TypeError: unsupported operand type(s) for +: 'NoneType' and 'str'```

**Comment by ArnaudBelcour** [comment]
Fixed in e3035b9afb7984d0b62a2e9acdedf0c7776196b2.

**Comment by ArnaudBelcour** [comment]
Fixed in #62.
